tests/e2e/configurer/upgrade.go

```diff
@@ -124,6 +127,19 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 
 	chainANode.CreatePool("pool1A.json", initialization.ValidatorWalletName)
 	chainBNode.CreatePool("pool1B.json", initialization.ValidatorWalletName)
+
+	// enable superfluid assets on both chains
+	chainA.EnableSuperfluidAsset("gamm/pool/1")
```

Comment:
This creates a dependency for the superfluid tests on the upgrade tests. Now, I don't think we can the superfluid test with the upgrade disabled because superfluid asset is not enabled.

Can we keep this logic in the test itself?